### ISSUE TITLE
R Code Injection/Code Reuse not working

### ISSUE THREAD

**Issue by camccaffrey** [twibiral/obsidian-execute-code#84]
If I put the following code in an R code block, it will output a scatterplot as expected.
```r
library(ggplot2)
scatterplot <- ggplot() +
	geom_point(aes(x=rnorm(10),y=rnorm(10)))
plot(scatterplot)
```

However, if I move the `library(ggplot2)` statement to the "Inject R code" box in the plugin settings or to a `pre-r` code block, I get the following error message:
```
Error in ggplot() : could not find function "ggplot"
Execution halted
```

In this example, it seems as if the package cannot load unless I include it in the original code block. Is this a bug, or am I using these two features incorrectly?

**Comment by milan338** [comment]
Thanks for pointing this out, it's a bug. I'm also getting the same issue, should have a fix soon.

**Comment by twibiral** [comment]
I fixed it and will release the fix in a few moments

**Comment by twibiral** [comment]
Fixed with release [0.18.0](https://github.com/twibiral/obsidian-execute-code/releases/tag/0.18.0). The code from the inject field was added but ignored then. This only happened for R.
